main.tf

```diff
@@ -5,4 +5,11 @@ provider "metal" {
 resource "metal_project" "new_project" {
   count = var.metal_create_project ? 1 : 0
   name  = var.metal_project_name
-}
\ No newline at end of file
+
+  # Kube-vip will enable BGP if not enabled, Terraform must match the settings
```

Comment:
This change may need to be applied sooner rather than later since subsequent `kubectl apply` commands will fail if this module created the EM project and kube-vip was applied.